packages/fxa-content-server/webpack.config.js

```diff
@@ -132,7 +132,13 @@ const webpackConfig = {
             loader: 'babel-loader',
             options: {
               cacheDirectory: true,
-              presets: ['@babel/preset-env'],
+              presets: [
+                [
+                  '@babel/preset-react', {
```

Comment:
Looks like this was easier to get included in the build process than I feared. Hooray!